packages-ts/gauntlet-cosmos-contracts/src/commands/contracts/ocr2/withdrawFunds.ts

```diff
@@ -30,7 +30,7 @@ const makeCommandInput = async (flags: any, args: string[]): Promise<CommandInpu
 
 const makeContractInput = async (input: CommandInput, context: ExecutionContext): Promise<ContractInput> => {
   const queryAvailableLink = async () =>
-    ((await context.provider.queryContractSmart(context.contract, 'link_available_for_payment' as any)) as any).amount
+    ((await context.provider.queryContractSmart(context.contract, {link_available_for_payment: {}} as any)) as any).amount
```

Comment:
a result of changing the query msg type from string to object with a string key (see first bullet point here for more context https://github.com/smartcontractkit/chainlink-cosmos/pull/359)